esp32c3-hal/.cargo/config.toml

```diff
@@ -1,5 +1,5 @@
 [target.riscv32imc-unknown-none-elf]
-runner = "espflash flash --monitor"
+runner = "espflash --monitor"
```

Comment:
In the whole repo we use the `espflash 2.x` syntax already - so would you mind chaning this back to `espflash flash --monitor`

I know the situation regarding `espflash 1.x` vs `espflash 2.x` is quite annoying and confusing but at least in one repo we should be consistent

Reply:
I was using `espflash 1.x` so it wasn't needed and I push it in the commit unintentionaly. I'll remove it.